pkg/translator/prometheusremotewrite/helper.go

```diff
@@ -44,7 +44,7 @@ const (
 
 var exportCreatedMetricGate = featuregate.GlobalRegistry().MustRegister(
```

Comment:
```
panic: no removal version set for Stable gate "exporter.prometheusremotewriteexporter.deprecateCreatedMetric"
```

Reply:
![image](https://github.com/user-attachments/assets/a6c0c821-389a-4c90-8628-f242e8a846f9)

Fixed [b2dea51](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/37399/commits/b2dea515931109ea16c502731a20f8099b12baa6)

Reply:
uhmm. 

[build-and-test / integration-tests-matrix (pkg) (pull_request)](https://github.com/open-telemetry/opentelemetry-collector-contrib/actions/runs/12914047871/job/36012766918?pr=37399)Failing after 3m

> "exporter.prometheusremotewriteexporter.deprecateCreatedMetric" is stable, can not be disabled

Taking a look...